week-2/errors/0.js

```diff
@@ -7,3 +7,4 @@ function capitalise(str) {
   let str = `${str[0].toUpperCase()}${str.slice(1)}`;
   return str;
 }
+/*"str" has already been declared, you don't need to use the key word "let".*/
```

Comment:
You have correctly identified the problem with the code.
The aim of this section is to get familiar with [error messages in Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors). To that end, what error message do you think will come up when you run this code? What error message was actually displayed?